packages/mermaid/src/diagrams/git/gitGraphAst.ts

```diff
@@ -444,13 +510,13 @@ export const prettyPrint = function () {
 export const clear = function () {
   commits = {};
   head = null;
-  let mainBranch = getConfig().gitGraph.mainBranchName;
-  let mainBranchOrder = getConfig().gitGraph.mainBranchOrder;
+  const mainBranch = getConfig().gitGraph?.mainBranchName ?? defaultConfig.gitGraph.mainBranchName;
+  const mainBranchOrder = getConfig().gitGraph?.mainBranchOrder ?? defaultConfig.gitGraph.mainBranchOrder;
```

Comment:
It's good to reduce calls to `getConfig()`. Storing the result into a local variable is a good option.